src/PollingBlockTracker.ts

```diff
@@ -54,15 +55,21 @@ export class PollingBlockTracker extends BaseBlockTracker {
     return await this.getLatestBlock();
   }
 
-  protected _start(): void {
-    this._synchronize().catch((err) => this.emit('error', err));
+  protected async _start(): Promise<void> {
+    await super._start();
```

Comment:
Hmm. This function is a no-op. Any reason you added it here?

Reply:
The coverage reporter was complaining that the method in the base class wasn't covered without this, so I added this. That said, I'm wondering if we can solve this problem a different way.

Reply:
Yeah, if I remove this, then the coverage reporter complains that BaseBlockTracker is not fully covered. I mean... it's not wrong. And I'm not sure if Jest runs files through TypeScript before passing them off to Istanbul, but if so, then Istanbul doesn't know that BaseBlockTracker is a base class.

Thoughts?